Society for Neuroscience Abstract 2016.js

```diff
@@ -0,0 +1,471 @@
+	} //else if (url.search(/(presentations|session|participant)\/[^\\]+\/\d+/) != -1) {
+	//	return 'multiple'; // does not work
+	//}
+}
+
+function doWeb(doc, url) {
+
+	if (detectWeb(doc,url) == 'conferencePaper') {
+
+		scrape(doc,url);
+
+	}
+	//TODO does not work properly becuase the page HTML is dynamically generated and
+	// ZU.processDocuments() cannot obtain the actual page content from URL.
+}
+
+
+
+
+function scrape(doc,url){
+
+	// NOTE doGet() html does not contain the actual contents,
+	// object is HTTPreqest object, use processDocuments
+	//
+	// Zotero.debug(doc); //object HTMLDocument
+	// Zotero.debug(doc.URL);
+
+	var item = new Zotero.Item("conferencePaper");
+
+	// pages, title
+	var h1color_primary = ZU.xpath(doc,'//h1[@class="color-primary"]');
+
+	//TODO when multiple doc does not contain the contents
+
+	// Zotero.debug(h1color_primary);
+	//TODO does not work for mulitiple
+
+	var m1 = h1color_primary[0].innerHTML.match(/^[.\n\s\d\w\/\-]+(?=\s-\s)/);
+	if (m1) {
+		item.pages = m1[0].replace(/\s|\n/g,"");
+	}
+
+	//Zotero.debug(item.pages);
+
+	var ogtitle = ZU.xpath(doc,'//meta[@property="og:title"]');
+	item.title = ZU.xpath(ogtitle,'@content')[0].value;
+
+	//Zotero.debug(item.pages);
+	//Zotero.debug(item.title);
+
+
+	// extra
+	var well =ZU.xpath(doc,'//div[@class="well well-small"]');
+	//Zotero.debug(well[0].innerHTML);
+
+	var mst = well[0].innerHTML.match(/<dt>Session Type<\/dt>[.\n\s]*<dd>(.+)<\/dd>/);
+
+	if (mst == null) {
+		mst = "";
+	}
+
+	//Zotero.debug(mst);
+	var session = ZU.xpath(doc,'//h2[@class="session"]/a');
+
+	item.extra = "Session: " + session[0].innerText
+		+ "; " + "Session Type: " + mst[1];
+
+
+	//attachments
+	//TODO does not work
+	item.attachments.push({
+		title: "Snapshot",
+		document: doc
+	});
+
+
+	// creators
+
+	// var disclosure = ZU.xpath(doc,'//div[@class="span7"]/dl/dd[2]'); // Use Disclosure data
+
+	// var m = disclosure[0].innerHTML.match(/<b>[^<]+:<\/b>/g);
+	// var names = [];
+	// for (i = 0; i < m.length; i++) {
+	// 	item.creators.push(ZU.cleanAuthor(m[i].match(/<b>([^<]+):<\/b>/)[1],"author"));
+	// }
+
+	var disclosures = ZU.xpath(doc, '//dl/dt[contains(., "Disclosures")]/following-sibling::dd');
+	var authorsList = ZU.xpath(doc, '//dl/dt[contains(., "Authors")]/following-sibling::dd');
+	if (disclosures) {
+		var authors = ZU.xpath(disclosures[0], './b');
+	} else if (authorsList) {
+		var authors = authorsList.split(',');
+	}
+	if (authors) {
+		for (var i=0; i<authors.length; i++) {
+			item.creators.push(ZU.cleanAuthor((authors[i].textContent), "author"));
+		}
+	}
+
+	Zotero.debug(item.creators);
+
+
+	// date
+	var spandate = ZU.xpath(doc,'//span[@class="session-date"]');
+
+	item.date = spandate[0].innerHTML.match(/\w+\s\d+,\s\d+/)[0];
+
+	if (ZU.xpath(doc,'//div[@class="span7"]/dl/dd[3]').length != 0) {
+		item.abstractNote = ZU.xpath(doc,'//div[@class="span7"]/dl/dd[3]')[0].innerHTML;
+	}
+
+	item.publicationTitle = "Society for Neuroscience Abstract";
```

Comment:
I realized that this value needs to be captured from the web page to make this translator generic.

However, it appears that there is no fixed element that always holds the name of the conference or the name of the abstracts.

```JavaScript
ZU.xpath(doc,'//div[@id="top-header"]//div[@class="top-matter"]/a')[0].innerText;
```
will return "Neurosceice 2017 Home" or "ACC Scientific Session Program Planner Home" or "AACR Annual Meeting 2017 Online Proceedings and Itinerary Planner Home" dependent on conference. None of these is the proper name for the conference.